ui/compontents.go

```diff
@@ -336,6 +342,212 @@ func endToEndRow(gtx layout.Context, leftWidget, rightWidget func(C) D) layout.D
+						pointer.Rect(image.Rectangle{Max: gtx.Constraints.Max}).Add(gtx.Ops)
+						click.Add(gtx.Ops)
+						page.walletAccountsHandler(gtx, wallAcctSelector.walletAccounts[windex][aindex])
+						return page.walletAccountLayout(gtx, wallAcctSelector.walletAccounts[windex][aindex])
+					})
+				})
+			})
+		},
+	}
+
+	return wallAcctSelector.walletAccount.Layout(gtx, w, 850)
+}
+
+func (page *pageCommon) walletAccountsHandler(gtx layout.Context, wallAcct walletAccount) {
+	for _, e := range wallAcct.evt.Events(gtx) {
+		if e.Type == gesture.TypeClick {
+			*page.selectedWallet = wallAcct.walletIndex
+			*page.selectedAccount = wallAcct.accountIndex
+			page.wallAcctSelector.isWalletAccountModalOpen = false
+		}
+	}
+}
+
+func (page *pageCommon) walletAccountLayout(gtx layout.Context, wallAcct walletAccount) layout.Dimensions {
+	accountIcon := page.icons.accountIcon
+	accountIcon.Scale = 1
+	inset := layout.Inset{
+		Bottom: values.MarginPadding10,
+	}
+	return inset.Layout(gtx, func(gtx C) D {
+		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				return layout.Flex{}.Layout(gtx,
+					layout.Flexed(0.1, func(gtx C) D {
+						inset := layout.Inset{
+							Right: values.MarginPadding10,
+							Top:   values.MarginPadding15,
+						}
+						return inset.Layout(gtx, func(gtx C) D {
+							return accountIcon.Layout(gtx)
+						})
+					}),
+					layout.Flexed(0.8, func(gtx C) D {
+						return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+							layout.Rigid(func(gtx C) D {
+								accountLabel := page.theme.Body2(wallAcct.accountName)
+								accountLabel.Color = page.theme.Color.Text
+								accountBalLabel := page.theme.Body2(wallAcct.totalBalance)
+								accountBalLabel.Color = page.theme.Color.Text
+								return page.accountTableLayout(gtx, accountLabel, accountBalLabel)
+							}),
+							layout.Rigid(func(gtx C) D {
+								spendibleLabel := page.theme.Body2("Spendable")
+								spendibleLabel.Color = page.theme.Color.Gray
+								spendibleBalLabel := page.theme.Body2(wallAcct.spendable)
+								spendibleBalLabel.Color = page.theme.Color.Gray
+								return page.accountTableLayout(gtx, spendibleLabel, spendibleBalLabel)
+							}),
+						)
+					}),
+					layout.Flexed(0.1, func(gtx C) D {
+						inset := layout.Inset{
+							Right: values.MarginPadding10,
+							Top:   values.MarginPadding10,
+						}
+
+						if *page.selectedWallet == wallAcct.walletIndex && *page.selectedAccount == wallAcct.accountIndex {
+							return layout.E.Layout(gtx, func(gtx C) D {
+								return inset.Layout(gtx, func(gtx C) D {
+									return page.icons.navigationCheck.Layout(gtx, values.MarginPadding20)
+								})
+							})
+						}
+						return layout.Dimensions{}
+					}),
+				)
+			}),
+		)
+	})
+}
+
+func (page *pageCommon) accountTableLayout(gtx layout.Context, leftLabel, rightLabel decredmaterial.Label) layout.Dimensions {
```

Comment:
This looks like it does the same thing as the "endToEnd" layout. Have a look at the "endToEnd" function in components.go and see how it is used.